{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py

```diff
@@ -2,7 +2,9 @@
 Django settings for {{cookiecutter.django_project_name}} project.
 """
 
+{% if cookiecutter.use_celery == "y" %}
 from datetime import timedelta
+{% endif %}
```

Comment:
You judiciously added dashes to remove unneeded empty lines everywhere, which is very much appreciated. Why not here? Did the linter caught all the other ones?

Reply:
I worked on these lines just to ensure that the linter is not complaining. Completely forgot about this. Thank you!